packages/discovery-react-components/src/components/SearchResults/components/Result/Result.tsx

```diff
@@ -119,7 +119,11 @@ export const Result: React.FunctionComponent<ResultProps> = ({
   const hasText = displayedText && !showTablesOnlyResults;
   const emptyResultContent = !(hasText || tableHtml);
 
-  const title = getDocumentTitle(result, resultTitleField);
+  let title = getDocumentTitle(result, resultTitleField);
+
+  if (Array.isArray(title)) {
+    title = title.join(', ');
```

Comment:
this doesn't make sense - why would we be joining the titles with commas? shouldn't we just select the first element? commas are not i18n-friendly for other languages

Reply:
Honestly, I think just showing the first element is the best choice, but I was never really given any clear answer on what should be displayed when I asked about this.  Granted this is very much an edge case where someone would deliberately annotate several title fields and then search by the title field.

I can ask again to hopefully get a clear answer.

Reply:
I agree with the maniac: don't do this; just select the first one.

Reply:
This was the conclusion we reached in standup, will add this change.